docs/versioned_docs/version-2.0.0-beta/setup/google-cloud-run.md

```diff
@@ -22,30 +23,41 @@ Follow the steps below to deploy ToolJet on Cloud run with `gcloud` CLI.
    docker push gcr.io/<replace-your-project-id>/tooljet/tooljet-ce:latest
    ```
 
-2. Deploy new cloud run service
+  Please run the above command by launching GoogleCLI which will help to push the Tooljet application image to Google container registry. 
 
-:::info
-This command takes the assumption that certain required environment has already been created in secrets. If you haven't created already then use the [secret manager](https://console.cloud.google.com/security/secret-manager).
-:::
+  <div style={{textAlign: 'center'}}>
+  <img className="screenshot-full" src="/img/cloud-run/CLI.png" alt="CLI" />
+  </div>
 
-   ```bash
-   gcloud run deploy tooljet-ce --image gcr.io/<replace-your-project-id>/tooljet/tooljet-ce:latest  \
-   --allow-unauthenticated \
-   --cpu 1 \
-   --memory 1Gi \
-   --min-instances 1 \
-   --set-env-vars "TOOLJET_HOST=https://<replace-your-public-host>.com" \
-   --set-secrets "PG_HOST=PG_HOST:latest" \
-   --set-secrets "PG_DB=PG_DB:latest" \
-   --set-secrets "PG_USER=PG_USER:latest" \
-   --set-secrets "PG_PASS=PG_PASS:latest" \
-   --set-secrets "LOCKBOX_MASTER_KEY=LOCKBOX_MASTER_KEY:latest" \
-   --set-secrets "SECRET_KEY_BASE=SECRET_KEY_BASE:latest" \
-   --args "npm,run,start:prod"
-   ```
 
-Update `TOOLJET_HOST` environment variable if you want to use the default url assigned with Cloud run after the initial deploy.
+2. Create new cloud run service
+
+	Select and add the pushed Tooljet application image as shown below.
+
+  <div style={{textAlign: 'center'}}>
+  <img className="screenshot-full" src="/img/cloud-run/tooljet-app-service.png" alt="tooljet-app-service" />
+  </div>
+
+3. Ingress and Authentication can set as shown below. But also can be changed to your preferred security options.
```

Comment:
Ingress and Authentication can be set as shown below, to begin with. Feel free to change the security configurations as per you see fit.